js/renovation/ui/box/box.tsx

```diff
@@ -3,8 +3,14 @@ import {
 } from '@devextreme-generator/declarations';
 import { Widget } from '../common/widget';
 import { BoxProps } from './box_props';
+import { combineClasses } from '../../utils/combine_classes';
 
-export const viewFunction = (): JSX.Element => (<Widget />);
+export const viewFunction = (): JSX.Element => {
+  const cssClasses = combineClasses({ 'dx-box dx-box-flex': true });
```

Comment:
viewFunction should be clear as possible. All operation that should be execute we should transfer to getters.

Reply:
Yep, fixed